esp/platform/txsummary.cpp

```diff
@@ -591,6 +603,15 @@ void CTxSummary::log(const LogLevel logLevel, const unsigned int requestedGroup,
             serialize(summary.clear(), logLevel, requestedGroup, TXSUMMARY_OUT_JSON);
             DBGLOG("%s", summary.str());
         }
+
+        if (requestedStyle & TXSUMMARY_FWD_OTEL)
+        {
+            // String values have already been forwarded and should ignore this serialization
+            // request. Timer values have not been forwarded, and should forward their values.
+            otelConnector = connector.get();
+            serialize(summary.clear(), logLevel, requestedGroup, TXSUMMARY_FWD_OTEL);
+            otelConnector = nullptr;
```

Comment:
what's the lifespan of the "connectors" one per thread/request?

Reply:
The connector exists for the duration of the CTxSummary, which is basically the duration of the the ESP context. This method is triggered from the context destructor.